### PR TITLE
TASK-49334_49335 : Implement wallet backup and wallet restore

### PR DESCRIPTION
**Pull request by GouadriaHanen**
TASK-49334: add Backup your wallet to wallet settings
TASK-49335: Restore your wallet drawer for all users since it was just for users who do not have a wallet.

### CODE REVIEW

**Review by ahamdi — APPROVED** [review]

**Review by ahamdi — CHANGES_REQUESTED** [review]
Could you add a PR description

File: wallet-webapps-common/src/main/webapp/vue-app/components/WalletImportKeyDrawer.vue [hunk 1]
```diff
@@ -36,28 +38,37 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           ">
           <v-text-field
             v-model="walletPrivateKey"
-            :append-icon="walletPrivateKeyShow ? 'mdi-eye' : 'mdi-eye-off'"
             :rules="[rules.priv]"
-            :type="walletPrivateKeyShow ? 'text' : 'password'"
+            :type="password"
```
Comment (ahamdi) [inline wallet-webapps-common/src/main/webapp/vue-app/components/WalletImportKeyDrawer.vue:42]: shouldn't we keep the verification of the private key value by displaying/hiding the content ?

File: wallet-webapps-common/src/main/webapp/vue-app/wallet-settings/components/WalletSettingsDetails.vue [hunk 2]
```diff
@@ -30,47 +30,47 @@
...
-                  :title="$t('exoplatform.wallet.title.addressQRCode')" />
-                <wallet-reward-address :value="wallet.address" :allow-edit="false" />
-              </v-list-item-action>
-            </v-list-item>
-          </div>
-          <v-list-item v-if="!wallet.address" class="mx-5 my-3">
+          <v-list-item>
+            <v-list-item-content>
+              <v-list-item-title class="title text-color">
+                {{ $t('exoplatform.wallet.label.mangePassword') }}
```
Comment (ahamdi) [inline wallet-webapps-common/src/main/webapp/vue-app/wallet-settings/components/WalletSettingsDetails.vue]:
```suggestion
                {{ $t('exoplatform.wallet.label.managePassword') }}
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: wallet-webapps-common/src/main/webapp/vue-app/components/WalletImportKeyDrawer.vue (hunk 1 above)
Comment (GouadriaHanen) [reply wallet-webapps-common/src/main/webapp/vue-app/components/WalletImportKeyDrawer.vue:42]: discussed with @mzorai  to make it visible

Comment (ahamdi) [reply wallet-webapps-common/src/main/webapp/vue-app/components/WalletImportKeyDrawer.vue:42]: Ok then the private key will be always hidden, and not visible

**Review by ahamdi — APPROVED** [review]

**Pull request merged**